docs/content/en/docs/Tutorials/share-tutorial.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Try out the share functionality in Reva"
+linkTitle: "Share functionality"
+weight: 5
+description: >
+  Try the share functionality in Reva locally.
+---
+
+This is a guide on how to try the share functionality in Reva in your local environment.
+
+## Prerequisites
+* golang
+* make/automake
+* git
+* curl or wget
+
+## 1. Clone the Reva repos
+Clone the reva repo from https://github.com/cs3org/reva 
+
+```
+git clone https://github.com/cs3org/reva
+```
+
+## 2. Build Reva
+Go to your Reva folder 
+
+```
+cd reva
+```
+
+Now you need to build Reva by running the following commands (you need to be in the *reva* folder)
+
+```
+make deps
+```
+
+```
+make build-reva
+```
+
+```
+make build-revad
+```
+
+## 4. Run Reva
+Now you need to run Revad (the Reva deamon). Follow these steps
+from the *reva* folder:
+
+```
+cd examples/ocmd/
+``` 
+
+```
+../../cmd/revad/revad -dev-dir .
+``` 
+
+The Reva daemon (revad) should now be running.
+
+## 5. Prepare an example file
+Open a new terminal and go to your reva folder.
+
+```
+echo "Example file" > example.txt
+```
+
+## 6. Log in to reva
+```
+./cmd/reva/reva login basic
+```
+
+If you now get an error saying that you need to run reva configure, do as follows:
+Run:
+
+```
+./cmd/reva/reva configure
+```
+
+and use 
+
+```
+host: 127.0.0.1:19000
+```
+
+Once configuered run:
+
+```
+./cmd/reva/reva login basic
+```
+
+And use the following log in credentials:
+
```

Comment:
It seems like loggin in is not necessary for testing the shares, @ishank011 and @JarCz.

Reply:
For uploading the file, you need to login. This login works for reva operations, which you perform using ./cmd/reva/reva. When you send a request, it doesn't know anything about your cmd configuration, so you have to send the auth header.

Reply:
HTTP "/shares" endpoints use http basic auth to get access. You can generate new "auth header" using software like "Postman" or change curl request to 
```
curl --request GET \
  --url http://127.0.0.1:19001/ocm/shares \
  --user login:password
```

Reply:
Ok, thank you for the clarification!